src/MAVN.Service.PartnerManagement.DomainServices/MAVN.Service.PartnerManagement.DomainServices.csproj

```diff
@@ -6,6 +6,7 @@
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="9.0.0" />
     <PackageReference Include="geohash-dotnet" Version="1.0.4" />
+    <PackageReference Include="Lykke.Common" Version="7.5.0" />
```

Comment:
This is not needed , cause this nugget package already implicitly referenced from other Lykke libs